packages/@ember/-internals/glimmer/lib/helper.ts

```diff
@@ -183,6 +187,10 @@ class ClassicHelperManager implements HelperManager<ClassicHelperStateBucket> {
   }
 }
 
+function isFactoryManager<T, C>(obj: unknown): obj is FactoryManager<T, C> {
+  return obj != null && 'class' in (obj as FactoryManager<T, C>);
```

Comment:
is a `.class` property the _only_ requirement for calling `create` on `obj`?

Reply:
https://github.com/emberjs/ember.js/blob/master/packages/@ember/-internals/container/lib/container.ts#L250-L253

Reply:
for extra certainty, should this function also check that create exists and is a function?